spec/controllers/spree/admin/users_controller_spec.rb

```diff
@@ -10,23 +10,22 @@
     before do
       allow(controller).to receive_messages spree_current_user: user
       allow(Spree::User).to receive(:find).with(test_user.id.to_s).and_return(test_user)
-      user.spree_roles.clear
     end
 
     it 'should grant access to users with an admin role' do
-      user.spree_roles << Spree::Role.find_or_create_by(name: 'admin')
+      user.update!(admin: true)
       spree_post :index
       expect(response).to render_template :index
     end
 
     it "allows admins to update a user's show api key view" do
-      user.spree_roles << Spree::Role.find_or_create_by(name: 'admin')
+      user.update!(admin: true)
       spree_put :update, id: test_user.id, user: { show_api_key_view: true }
       expect(response).to redirect_to spree.edit_admin_user_path(test_user)
     end
 
     it "re-renders the edit form if error" do
-      user.spree_roles << Spree::Role.find_or_create_by(name: 'admin')
+      user.update!(admin: true)
       spree_put :update, id: test_user.id, user: { password: "blah", password_confirmation: "" }
 
       expect(response).to render_template :edit
```

Comment:
3 out of 4 tests require an admin user, so we could set it up as an admin user in the beginning.
In fact, the fourth test mocks the admin? method anyway..
Maybe I'll submit a commit.